### PR TITLE
Add validation message for repoInput

### PR DESCRIPTION
**Pull request by shaedrich**
Follow-up to #8:

> Thanks, merged! In terms of UX, it would be better to (also) show an error, since just disabling controls is confusing, but this is better than nothing.

_Originally posted by @LeaVerou in https://github.com/LeaVerou/issue-closing/issues/8#issuecomment-1701321632_
            

Maybe, it would be more precise if we tested for exactly one occurance of a slash only: `(repoInput.match(/\//g)||[]).length === 1` or checked the whole `repoInput` via RegExp: `repoInput.test(/[a-z]+\/[a-z]+/)` (is that the closest and correctest we can get?)

Would you also like a message to show up when the validation is successful?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +15,9 @@
 		<header>
 			<h1><a href="?repo=[repo]" title="Click for permalink">GitHub Issue Closing Times</a></h1>
 			<form mv-action="set(repo, repoInput), set(labels, labelFilter)">
+				<p class="mv-message mv-warning mv-inline" mv-if="!(len(repoInput) >= 3 and contains(repoInput, '/'))">
+					The entered repository name is not value. Please enter it in the format <code>organisation/repository</code>
```
Comment (LeaVerou) [inline index.html]:
```suggestion
					The entered repository name is not valid. Please enter it in the format <code>organisation/repository</code>
```

**Comment by LeaVerou** [comment]
> Follow-up to #8
> 
> Maybe, it would be more precise if we tested for exactly one occurance of a slash only: `(repoInput.match(/\//g)||[]).length === 1` or checked the whole `repoInput` via RegExp: `repoInput.test(/[a-z]+\/[a-z]+/)` (is that the closest and correctest we can get?)

I don't think that's necessary, we can see if it comes up.

> Would you also like a message to show up when the validation is successful?

Nah, that would be extra clutter. Data being valid is the default :)

**Comment by shaedrich** [comment]
> Nah, that would be extra clutter. Data being <del>value</del><ins>valid</ins> is the default :)

The typo is fixed&mdash;is there anything else, I can adjust before you merge?

File: index.html [hunk 2]
```diff
@@ -15,6 +15,9 @@
 		<header>
 			<h1><a href="?repo=[repo]" title="Click for permalink">GitHub Issue Closing Times</a></h1>
 			<form mv-action="set(repo, repoInput), set(labels, labelFilter)">
+				<p class="mv-message mv-warning mv-inline" mv-if="!(len(repoInput) >= 3 and contains(repoInput, '/'))">
+					The entered repository name is not valid. Please enter it in the format <code>organisation/repository</code>
```
Comment (LeaVerou) [inline index.html]:
```suggestion
					The entered repository name is not valid. Please enter it in the format <code>owner/repository</code>
```

"Owner" encompasses both users and orgs

**Review by LeaVerou — CHANGES_REQUESTED** [review]
Tiny nit, then it's good to go

**Comment by shaedrich** [comment]
Alright! Done :)

**Review by LeaVerou — APPROVED** [review]

**Pull request merged**